demos/action_recognition_demo/python/action_recognition_demo.py

```diff
@@ -88,34 +88,34 @@ def main():
 
     log.info('OpenVINO Inference Engine')
     log.info('\tbuild: {}'.format(get_version()))
-    ie = IECore()
+    core = Core()
 
     if 'MYRIAD' in args.device:
         myriad_config = {'VPU_HW_STAGES_OPTIMIZATION': 'YES'}
-        ie.set_config(myriad_config, 'MYRIAD')
+        core.set_config(myriad_config, 'MYRIAD')
 
     if args.cpu_extension and 'CPU' in args.device:
-        ie.add_extension(args.cpu_extension, 'CPU')
+        core.add_extension(args.cpu_extension, 'CPU')
```

Comment:
I think we need to remove extensions, as we do not have models with extended layers

Reply:
Removed.